register/urls.py

```diff
@@ -4,9 +4,15 @@
 
 
 urlpatterns = [
+    url(r'^logout/$', views.LogoutView.as_view(), name='auth_logout'),
+    url(r'^login/$', views.EmailView.as_view(), name='auth_email'),
+    # Keep 'registration.backends' URLs after '/login/' and before 'auth_login'
+    # for proper overriding.
+    url(r'^', include('registration.backends.default.urls')),
+    url(r'^auth/login/$', views.LoginView.as_view(), name='auth_login'),
```

Comment:
I know. I'm not sure how to deal with this otherwise. The idea is to have a very minimal url for the app to login (like here `/login`) and then to have routes that match what they do (actual `login` and `register`). I used `signin` earlier but I find that action so confusing.